app/src/main/java/org/oppia/app/walkthrough/WalkthroughFragmentPresenter.kt

```diff
@@ -15,18 +17,39 @@ class WalkthroughFragmentPresenter @Inject constructor(
 ) {
   private lateinit var binding: WalkthroughFragmentBinding
   private var currentProgress: Int = 0
+  private lateinit var viewPager: ViewPager2
 
   fun handleCreateView(inflater: LayoutInflater, container: ViewGroup?): View? {
     binding = WalkthroughFragmentBinding.inflate(inflater, container, /* attachToRoot= */ false)
 
     binding.let {
       it.lifecycleOwner = fragment
       it.presenter = this
+      viewPager = binding.root.findViewById(R.id.walkthrough_view_pager) as ViewPager2
+
     }
+    setUpViewPager(viewPager)
+
 
     return binding.root
   }
 
+  private fun setUpViewPager(viewPager: ViewPager2) {
+    val adapter = WalkthroughPagerAdapter(fragment.requireActivity())
+    viewPager.apply {
+      this.adapter = adapter
+      isUserInputEnabled = false
+
```

Comment:
ditto. Please remove this empty line also.

Reply:
done